### PR TITLE
Adds additional notes on tools

### PR DESCRIPTION
**Pull request by SteveALee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,7 +208,7 @@ For the uninitiated, MAX_PATH is a limitation with many Windows tools and APIs t
   moves duplicate packages to top-level
 * `> npm install -g flatten-packages`
   moves all packages to top-level, but can cause versioning issues
-* :heart: Upgrade to npm@3
+* :heart: Upgrade to npm@3 which flattens out dependencies and so avoids nested node_module folders.
```
Comment (mousetraps) [inline README.md]: let's be more precise on this - npm@3 makes things *maximally flat* - you can still run into issues when versions conflict.

Comment (SteveALee) [inline README.md]: Good point. will do with the rebase

**Comment by SteveALee** [comment]
@mousetraps  this and #18 now reworked and ready to merge.

**Comment by mousetraps** [comment]
:+1:

**Pull request merged**